fixtures/attribute-behavior/src/attributes.js

```diff
@@ -356,7 +356,12 @@ const attributes = [
   },
   {name: 'cols', tagName: 'textarea'},
   {name: 'colSpan', containerTagName: 'tr', tagName: 'td'},
-  {name: 'content', tagName: 'meta'},
+  {
+    name: 'content',
+    containerTagName: 'head',
+    tagName: 'meta',
+    extraProps: {name: 'description'},
```

Comment:
Do you want to remove the extra props to assert the tag still shows up with it has minimal props?

Reply:
Definitely if it works now.

Reply:
Removed the `extraProps` since the fixture works without them. Thank you for pointing this out!